smarthost/etc/e-smith/templates/etc/postfix/tls_policy/10smarthost-policy

```diff
@@ -2,18 +2,35 @@
 # SmartHost policy
 #
 {
-    if($postfix{SmartHostStatus} ne 'enabled') {
-	return '';
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro('smarthosts');
+    my $dbc = esmith::ConfigDB->open_ro();
+
+    my $SmartHostName = $dbc->get_prop('postfix','SmartHostName') || '';
+    my $SmartHostPort = $dbc->get_prop('postfix','SmartHostPort') || '';
+    my $SmartHostTlsStatus = $dbc->get_prop('postfix','SmartHostTlsStatus') || '';
+    my $SmartHostStatus = $dbc->get_prop('postfix','SmartHostStatus') || 'disabled';
```

Comment:
The configuratio database is already accessible from templates, so why reopening it?
Example:
```
my $SmartHostName = $postfix{'SmartHostName'} || '';
```

Reply:
a bad habit I presume, it is  fixed